src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -189,9 +189,14 @@ private static void assertResultsEquals(ShortCircuitAnalysisResult result, org.g
 
     private static void assertPagedFaultResultsEquals(ShortCircuitAnalysisResult result, List<org.gridsuite.shortcircuit.server.dto.FaultResult> faultResults) {
```

Comment:
assertResultsEquals and assertPagedFaultResultsEquals are almost the same. One should be deleted